src/geo-utils.ts

```diff
@@ -135,12 +148,17 @@ export function levelNameFromElementId(id: string): string {
 }
 
 export function getAtlas(countryCode: string): UsAtlas {
-  if (countryCode == "US") {
-    return us as unknown as UsAtlas;
-  } else if (countryCode == "GB") {
-    return gb as unknown as UsAtlas;
-  } else if (countryCode == "IE") {
-    return ie as unknown as UsAtlas;
+  switch (countryCode) {
+    case "US":
+      return us as unknown as UsAtlas;
+    case "GB":
+      return gb as unknown as UsAtlas;
+    case "CA":
+      return ca as unknown as UsAtlas;
+    case "IE":
+      return ie as unknown as UsAtlas;
+    default:
+      console.log("Country atlas not found");
   }
```

Comment:
This switch statement doesn't scale.  Consider using a dictionary using a `Record` type paired with a `CountryCode` enum:
```
enum CountryCode {
  US = "US",
  GB = "GB",
  ...
}

/** Atlas data indexed by countryCode */
const ATLAS_BY_COUNTRY_CODE: Record<CountryCode, UsAtlas> = {
  US: us,
  GB: gb,
  ....
};
```

Reply:
Changed. coded seems to be working but visual studio is throwing this warning: 
Type '{ US: { type: string; bbox: number[]; transform: { scale: number[]; translate: number[]; }; objects: { counties: { type: string; geometries: ({ type: string; arcs: number[][]; id: string; properties: { name: string; }; } | { ...; })[]; }; states: { ...; }; nation: { ...; }; }; arcs: number[][][]; }; Gb: { ...; }; IE...' is not assignable to type 'Record<countryCode, UsAtlas>'.
  Object literal may only specify known properties, but 'US' does not exist in type 'Record<countryCode, UsAtlas>'. Did you mean to write 'us'?ts(2322)